test/avocado/parent.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import os
+
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+TEST_DIR = os.path.dirname(CURRENT_DIR)
+BOTS_DIR = os.path.join(os.path.dirname(TEST_DIR), "bots")
+LOCAL_COMMON_DIR = os.path.join(TEST_DIR, "common")
+LOCAL_COPY_COMMON_DIR = os.path.join(CURRENT_DIR, "common")
```

Comment:
What's the purpose of `LOCAL_COPY_COMMON_DIR` and `LOCAL_COPY_MACHINE_DIR`? This should at least get a comment. There should not be two copies of bots/machines/ and tests/common / anywhere?

Reply:
@martinpitt I'll add comment there. this is used inside run-test as location of libs on ``remote machine`` Now there were copied just ``avocado`` dir and thanks to symlinks it copyies also libs, but I've removed symlinks and replaced them by ``import parent`` so that testlib and testvm libraries are not there. so that I have to copy them